src/main/external-resources/nsis/setup.nsi

```diff
@@ -344,20 +344,25 @@ Section "Uninstall"
 
 	; Current renderer files
 	Delete /REBOOTOK "$INSTDIR\renderers\AnyCast.conf"
-	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-TV-VLC.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice-AirPlayer.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice-VLC32bit.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\Apple-iDevice.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\BlackBerry-PlayBook-KalemSoftMP.conf"
 	Delete /REBOOTOK "$INSTDIR\renderers\CambridgeAudio-AzurBD.conf"
+	Delete /REBOOTOK "$INSTDIR\renderers\DLink-DSM510.conf"
```

Comment:
"DL" shouldn't come before "De" in my opinion. I know some systems like to treat capitals differently but I don't find that as intuitive.